packages/widget/src/hooks/useGetSourceBalance.ts

```diff
@@ -17,6 +17,9 @@ export const useGetSourceBalance = () => {
     address: sourceAccount?.address,
   });
 
+  // this is to support both tanstack query v4/v5
+  const cw20BalanceIsLoading = cw20Balance.isPending && cw20Balance.isFetching;
```

Comment:
I think the correct logic is `cw20Balance.isPending || cw20Balance.isFetching;`

Reply:
I took the logic from https://tanstack.com/query/latest/docs/framework/react/guides/disabling-queries#isloading-previously-isinitialloading

Reply:
they said it's cw20Balance.isPending && cw20Balance.isFetching;

Reply:
I see, approved